spk/duplicity/Makefile

```diff
@@ -0,0 +1,34 @@
+SPK_NAME = duplicity
+SPK_VERS = 0.7.03
+SPK_REV = 1
+SPK_ICON = src/duplicity.png
+
+DEPENDS = cross/librsync
+BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel cross/netifaces cross/wrapt cross/$(SPK_NAME)
+WHEELS = src/requirements.txt
+SPK_DEPENDS = "python>=2.7:gnupg>=2.1"
```

Comment:
Use the full version. `python>=2.7.8-9` for wheel support. Same for gnupg, which would probably be `gnupg>=2.1-1`